cgsmiles/graph_layout.py

```diff
@@ -53,16 +89,44 @@ def assign_angles(graph):
         and the reference angle value
     """
     triplets = find_triplets(graph)
+    cycles = nx.cycle_basis(graph)
```

Comment:
Consider adding a maximum ring size here

Reply:
is that possible in cycle basis? Or you mean scan by size and filter?

Reply:
Oh! No, it doesn't make sense for the cycle basis indeed. You could do nx.simple_cycles (with a size limit). You could argue that that makes more sense than the cycle basis. :shrug: